### DISCUSSION TITLE
MantineThemeProvider.tsx:11 Uncaught Error: @mantine/core: MantineProvider was not found in component tree, make sure you have it in your app

### DISCUSSION THREAD

**Discussion by npearson72** [jotaijs/jotai-devtools#144] [Q&A] [answered]
I'm seeing the following error when I click on any atom in Jotai DevTools panel.

Here's a reproducible repo: https://github.com/npearson72/jotai-devtools-test

How it started:
![Screenshot 2024-05-12 at 11 09 21 AM](https://github.com/jotaijs/jotai-devtools/assets/1323414/2ce86c4e-27fc-4bdd-be6e-75a884d72d72)

How it's going:
![Screenshot 2024-05-12 at 11 09 33 AM](https://github.com/jotaijs/jotai-devtools/assets/1323414/3f0d12db-f537-4e9a-bba1-4d6788d11f0e)

**Comment by arjunvegda** [comment]
Thanks for the repro! This is because the repo has a conflict between `@mantine/core` v6 and v7. Removing [these 
Mantine deps](https://github.com/npearson72/jotai-devtools-test/blob/master/package.json#L13-L15) in your repo should fix it.

**Comment by npearson72** [comment]
> Thanks for the repro! This is because the repo has a conflict between `@mantine/core` v6 and v7. Removing [these Mantine deps](https://github.com/npearson72/jotai-devtools-test/blob/master/package.json#L13-L15) in your repo should fix it.

@arjunvegda I included Mantine v6 because that's what I use in my real world project, and we haven't migrated yet to v7 because of how big of a change it it is from v6. I take it by your comment that Jotai DevTools does not support Mantine v6?

**Reply by arjunvegda** [reply, marked as answer]
That's right! In that case, I'd recommend pinning the Jotai DevTools version @ 0.8.0, until you're ready 🙌

**Reply by npearson72** [reply]
Makes sense. Although I do wonder if this might be a worthwhile discussion to have upstream, as the implication here is that Mantine is not an unobtrusive option for library developers.

Not familiar enough with [shadcn/ui](https://ui.shadcn.com/) but maybe this would be a better option?

**Reply by arjunvegda** [reply]
I agree, I'm in touch with Mantine's author to see if it's worth making Mantine more friendly for library developers. 

I'm open to contributions on migrating the DevTools to Shadcn or other alternatives that provide a similar set of components and are well maintained. I just don't have the bandwidth to do that myself at this point 😅

Mantine's author might also be open to contributions, so that's another option.

**Reply by npearson72** [reply]
@arjunvegda sounds good. It would be ideal if @rtivital can help resolve it upstream. If not, please let me know and I'll work on swapping Jotai DevTools to a different component lib.

**Reply by arjunvegda** [reply]
However, just for clarity, I'm not imagining backward-compatible support for v6, but more thinking of making Mantine v7+ more friendly for library authors. Things like allowing library authors to update the `mantine-*` classes, customizing the `color-scheme` key, etc. 

Here is a link to Martine's discord server if you're curious - https://discord.gg/wbH82zuWMN

**Reply by npearson72** [reply]
Sounds good. I'm simply proposing decoupling downstream consumers from library styling details, and can help if desired.
